ui/src/logs/actions/index.ts

```diff
@@ -595,7 +628,17 @@ export const fetchMoreAsync = (queryTimeEnd: string) => async (
   const params = [namespace, proxyLink, tableQueryConfig]
 
   if (_.every(params)) {
-    const query = buildBackwardLogQuery(
+    const queryTimeStart = await findOlderLowerTimeBounds(
+      queryTimeEnd,
+      newQueryConfig,
+      filters,
+      searchTerm,
+      proxyLink,
+      namespace
+    )
+
+    const query = await buildInfiniteScrollLogQuery(
```

Comment:
does this merit a type?

Reply:
as the functions have a return type I don't think so